### PR TITLE
Adds MakeCallWithEvents allowing to set the StatusCallback

### PR DESCRIPTION
**Pull request by flyandi**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 /bazel-*
 profile.out
+.idea/
```
Comment (kevinburke) [inline .gitignore:3]: This is not an appropriate place for individual-user-specific project files; use .git/info/exclude instead. See  https://github.com/golang/go/issues/21458 for more details.

**Comment by kevinburke** [comment]
Thanks for this. Unfortunately the API for this project is already pretty big and I'm hesitant to make it unnecessarily bigger. I am glad to see that you got this working and you are welcome to add your own wrapper method that makes the changes you suggest.

**Comment by flyandi** [comment]
I understand but the StatusCallback is pretty important - you can make a call but you won't get any events so you can detect hangup's, failures, etc. It seems that is an essential for this library in order to work properly. Just sending out a call might not cut it when u write a full Twilio service.

**Comment by kevinburke** [comment]
> It seems that is an essential for this library in order to work properly.

You can always set it by `calling client.Calls.Create` directly. Perhaps we could switch this to an Example of how to use that method.

**Pull request closed without merging**